packages/contentstack-import/src/import/modules-js/webhooks.js

```diff
@@ -98,6 +98,15 @@ module.exports = class ImportWebhooks {
         .catch(function (error) {
           // error while importing environments
           fileHelper.writeFileSync(webFailsPath, self.fails);
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).webhook !== undefined) {
+            if (JSON.parse(error.request.data).webhook.name) {
+              message_content_type =
+                ' Due to Webhook name - ' + JSON.parse(error.request.data).webhook.name;
```

Comment:
@cs-raj Can you please validate the error messages with the docs team?

Reply:
Validated with Varsha and made the changes